dashboard/db/migrate/20170426001230_add_pd_workshop_on_map_and_funded.rb

```diff
@@ -0,0 +1,6 @@
+class AddPdWorkshopOnMapAndFunded < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pd_workshops, :on_map, :boolean
+    add_column :pd_workshops, :funded, :boolean
```

Comment:
It isn't obvious (at least to me) what these columns mean, even after reading the Trello card and your (otherwise nice) PR summary. Mind adding a brief explanation of these fields in this file? Thanks!